CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+When contributing to this service,
+please open an issue on
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+or send a mail to `apisupport@onoffice.de`
+before making a change.
+
+## Bugs
+
+Bugs must be reported via the
+[GitHub issues](https://github.com/onOfficeGmbH/sdk/issues)
+Bug fixes and improvements can be provided via a Pull Request.
```

Comment:
I think you forgot to end the first sentence here after the link to the GitHub issues page. Maybe change the end of the sentence to something like "via the GitHub issues page". For me, there is currently just something missing for it to sound correct.